pupil-spa/src/assets/shared-styles/elements/_warm-up.scss

```diff
@@ -4,5 +4,15 @@
   color: $white;
   padding:.526315em .789473em .263157em;
   font-size: 1em;
-  font-weight: bold
+  font-weight: bold;
+}
+
+.practice-box {
+  color: $white;
+  background-color: $grey-2;
+  display: inline-block;
+  font-weight: bold;
+  padding: 6px 15px 6px 15px;
```

Comment:
are we doing em values in this app? if yes then we need to use pem()

Reply:
Do you mean like it's done a few lines above? 
```padding:.526315em .789473em .263157em;```
Then the answer is NO =) I've converted them using the pem function.